include/zephyr/sys/reboot.h

```diff
@@ -34,6 +34,22 @@ extern "C" {
  */
 extern FUNC_NORETURN void sys_reboot(int type);
 
+/**
```

Comment:
Nothing wrong with this line, but the commit message just says "TO BE DESCRIBED". This should be fixed.